specs/wp-signup-spec.js

```diff
@@ -168,14 +188,70 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 										} );
 									} );
 
-									test.describe( 'Step Eight: Can activate my account from an email', function() {
-										test.before( function() {
-											return this.emailClient = new EmailClient( signupInboxId );
+									test.describe( `Step ${stepNum}: Can not publish until email is confirmed`, function() {
+										stepNum++;
+
+										test.it( 'Can see a disabled publish button', function() {
+											const blogPostTitle = dataHelper.randomPhrase();
+											const blogPostQuote = dataHelper.randomPhrase();
+
+											this.adminTopBar = new AdminTopBar( driver );
```

Comment:
We shouldn't use the admin bar - just use the calypso base url to visit the post page and use it directly (or navigate to it via calypso) - this is so these can run on any calypso environment including `calypso.localhost:3000`